src/io/flutter/view/FlutterView.java

```diff
@@ -333,10 +339,27 @@ private void addPerformanceTab(JBRunnerTabs runnerTabs,
       runnerTabs.select(tabInfo, false);
     }
 
-    if (!selectedTab) {
-      assert app.getPerfService() != null;
-      app.getPerfService().pausePolling();
+//    if (!selectedTab) {
+//      assert app.getPerfService() != null;
+//      app.getPerfService().pausePolling();
+//    }
+  }
+
+  private void addMemoryTab(JBRunnerTabs runnerTabs,
+                                 FlutterApp app,
+                                 boolean selectedTab) {
+    final InspectorMemoryTab perfTab = new InspectorMemoryTab(runnerTabs, app);
+    final TabInfo tabInfo = new TabInfo(perfTab)
+      .append(MEMORY_TAB_LABEL, SimpleTextAttributes.REGULAR_ATTRIBUTES);
+    runnerTabs.addTab(tabInfo);
+    if (selectedTab) {
+      runnerTabs.select(tabInfo, false);
     }
+
+//    if (!selectedTab) {
```

Comment:
Add comment why you don't want to pause polling and remove the commented out code.

Reply:
Removed commented out code and implemented your suggestion of addPollingClient removePollingClient.  I suspect we might want to have profilingOn and profilingOff which does same thing but is toggled by a profiling button in the toolbar.  But this is for another discussion.